Add Pico 4 Pro/Enterprise via Pico Connect / Pico Business Streamer

Slime-Senpai opened the pull request:
I made a fork to support Pico 4 Pro / Enterprise.
Since I saw that you wanted people to contribute, I thought I should see if you wanted to add it to your wiki

Slime-Senpai committed:
Add Pico 4 Pro/Enterprise via Pico Connect / Pico Business Streamer

200Tigersbloxed commented:
This looks great! I assume you've already tested with VRCFT2CVR/Hypernex?

Slime-Senpai commented:
I've built this for CVR because someone with a Pico 4 Enterprise wanted to use it. So I've confirmed it works in CVR using the VRCFT2CVR plugin

200Tigersbloxed commented:
Just want to check in with the review, cause I'm not sure if it went through.

> Would it be possible to rename this to just `VRCFTPicoModule`? These modules are still generic to VRCFT and their only difference is just being built with a modified version of VRCFT. There should be nothing about the modules that are specific to Hypernex, VRCFT2CVR, etc. (excluding framework specifics).

Slime-Senpai commented:
I don't really have an opinion on this. I just thought I'd take the name of the module, if you prefer it more generic that's fine to me. I changed the name of the repo and updated it here. Also did you write the rename somewhere else than here ? I'm not very used to PR in github and I must say I can't find it anywhere x)

200Tigersbloxed commented:
This is excellent, thank you very much!

> Also did you write the rename somewhere else than here ? I'm not very used to PR in github and I must say I can't find it anywhere x)

Ah it was a review. Sometimes they just don't like to go through so I'm not sure,, ;p
